content/en/docs/Writing policies/jmespath.md

```diff
@@ -197,6 +195,16 @@ spec:
 
 To represent the `limits.memory` field in a JMESPath expression requires literal quoting of the key in order to avoid being interpreted as child nodes `limits` and `memory`. The expression would then be `{{ spec.hard.\"limits.memory\" }}`. A similar approach is needed when individual keys contain special characters, for example a dash (`-`). Quoting and then escaping is similarly needed there, ex., `{{ images.containers.\"my-container\".tag }}`.
 
+## Useful Patterns
+
+When developing policies for Kubernetes resources, there are several patterns which are common where JMESPath can be useful. This section attempts to capture example patterns that have been observed through real world use cases and how to write JMESPath for them.
+
+### Flattening Arrays
+
```

Comment:
Is this expected to be an empty section? (still WIP?)

Reply:
Yes lol

Reply:
...but now it's not.